lightningd/peer_htlcs.c

```diff
@@ -2109,32 +2109,29 @@ AUTODATA(json_command, &dev_ignore_htlcs);
 
 static void listforwardings_add_forwardings(struct json_stream *response, struct wallet *wallet)
 {
-	const struct forwarding *forwardings;
-	forwardings = wallet_forwarded_payments_get(wallet, tmpctx);
-
-	json_array_start(response, "forwards");
-	for (size_t i=0; i<tal_count(forwardings); i++) {
-		const struct forwarding *cur = &forwardings[i];
-		json_object_start(response, NULL);
-
-		json_add_short_channel_id(response, "in_channel", &cur->channel_in);
-		json_add_short_channel_id(response, "out_channel", &cur->channel_out);
-		json_add_amount_msat_compat(response,
-					    cur->msat_in,
-					    "in_msatoshi", "in_msat");
-		json_add_amount_msat_compat(response,
-					    cur->msat_out,
-					    "out_msatoshi",  "out_msat");
-		json_add_amount_msat_compat(response,
-					    cur->fee,
-					    "fee", "fee_msat");
-		json_add_string(response, "status", forward_status_name(cur->status));
-
-		if(cur->failcode != 0) {
-			json_add_num(response, "failcode", cur->failcode);
-			json_add_string(response, "failreason",
-					 onion_type_name(cur->failcode));
-		}
+	json_object_start(response, fieldname);
+
+	json_add_hex(response, "payment_hash",
```

Comment:
Why did you remove the `for` loop in **this** commit?

As-is if we bisect to this commit, compiling will result in failure as `cur` variable does not exist. 

You should probably remove the `for` loop on the *next* commit, and *only* add the `payment_hash` on *this* commit.

Reply:
Sorry, I squash the wrong commit.